tests/rados/scheduled_scrub_scenarios.py

```diff
@@ -0,0 +1,82 @@
+"""
+   This module contains the methods required for scrubbing.
+   1.To set the parameters for scrubbing initially required the
+     cluster time and day details.get_cluster_date method provides
+     the details.
+   2.set_osd_configuration method  used to set the configuration
+     parameters on the cluster.
+   3.get_osd_configuration  method is used to get the configured parameters
+     on the cluster.
+     NOTE: With set_osd_configuration & get_osd_configuration methods can
+          use to set the get the any OSD configuration parameters.
+   4. get_pg_dump  method is used to get the pg dump details from the cluster.
+   5. verify_scrub  method used for the verification of scheduled scrub.
+"""
+
+import logging
+import time
+from datetime import datetime
+
+from ceph.ceph_admin import CephAdmin
+from ceph.rados.rados_scrub import RadosScrubber
+
+log = logging.getLogger(__name__)
+
+
+def run(ceph_cluster, **kw):
+
+    OSD_SCRUB_MIN_INTERVAL = 1800
+    OSD_SCRUB_MAX_INTERVAL = 3600
+    OSD_DEEP_SCRUB_INTERVAL = 3600
```

Comment:
As they are inside the method, you can opt to use lower cases... The practice is to use upper cases only when it is global. This would not break anything though.

Reply:
Fixed the issue.